traffic_ops/traffic_ops_golang/crconfig/deliveryservice.go

```diff
@@ -76,6 +76,7 @@ inner join type as t on t.id = d.type
 left outer join profile as p on p.id = d.profile
 where d.cdn_id = (select id from cdn where name = $1)
 and d.active = true
+and t.name != 'ANY_MAP'
```

Comment:
Can you change this to `tc.DSTypeAnyMap`?

Reply:
done